### PR TITLE
[CPU] Disable inplace concat if any parent is constant

### PR DESCRIPTION
**Pull request by apertovs**
### Tickets:
 - *61352*

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp [hunk 1]
```diff
@@ -152,6 +152,11 @@ void MKLDNNConcatNode::initSupportedPrimitiveDescriptors() {
         }
     }
 
+    for (size_t i = 0; i < getParentEdges().size(); i++) {
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp:155]: Please add the comment why do we need this condition

**Comment by dmitry-gorokhov** [comment]
@apertovs We need to cover this case by tests. Is it possible to do that by subgraph one?

**Comment by dmitry-gorokhov** [comment]
@apertovs We also need internal validation to proof there is no perf regressions.

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp (hunk 1 above)
Comment (apertovs) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp:156]: Done

**Comment by apertovs** [comment]
> @apertovs We need to cover this case by tests. Is it possible to do that by subgraph one?

Yes, it's possible. Subgraph test added in latest version.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**